### ISSUE TITLE
Duplicate italian region

### ISSUE THREAD

**Issue by gdm85** [harpreetkhalsagtbit/country-state-city#19]
There are two regions for Italy, `Abruzzo` and `Abruzzi`, which should be only `Abruzzo`.

You can find the full list (20) here: https://it.wikipedia.org/wiki/Regioni_d%27Italia#Dati_demografici_e_geografici

**Comment by harpreetkhalsagtbit** [comment]
Wiki says this

In 1963 the Abruzzi and Molise region was again separated in the two regions of Abruzzo and Molise, bringing the current number of regions to 20.

https://it.wikipedia.org/wiki/Regioni_d%27Italia#Dati_demografici_e_geografici

**Comment by gdm85** [comment]
Yes, but there is no more "Abruzzi". You have to look at the current list.

**Comment by marcosantomo** [comment]
I'm also from Italy and I agree with @gdm85 
In my opinion, there are also regions and provinces both in the states field and this is a wrong representation.

**Comment by harpreetkhalsagtbit** [comment]
OK, I will consider your suggestion and will update it soon or better if you can send me a P.R

**Comment by sntM** [comment]
Just remove the 1802 state. By the way in order to have regions and provinces in two different lists there is the need to refactor all the project

________________________________
Da: Harpreet Singh <notifications@github.com>
Inviato: Tuesday, October 15, 2019 12:57:44 PM
A: harpreetkhalsagtbit/country-state-city <country-state-city@noreply.github.com>
Cc: sntM <marco.santomo@outlook.com>; Comment <comment@noreply.github.com>
Oggetto: Re: [harpreetkhalsagtbit/country-state-city] Duplicate italian region (#19)


OK, I will consider your suggestion and will update it soon or better if you can send me a P.R

**Comment by gdm85** [comment]
I have since then stopped using this library, it has plenty other issues; unsubscribing.
